lib/ffi-compiler/compile_task.rb

```diff
@@ -238,6 +254,10 @@ def cc
       def cxx
         @cxx ||= (ENV['CXX'] || RbConfig::CONFIG['CXX'] || 'c++')
       end
+
+      def go
+        @go ||= (ENV['GO'] || RbConfig::CONFIG['GO'] || 'go')
```

Comment:
Ruby doesn't define such a key: `RbConfig::CONFIG['GO']`

Reply:
Since it's in the stdlib, I also used `mkmf`'s [find_executable](https://ruby-doc.org/stdlib-2.5.1/libdoc/mkmf/rdoc/MakeMakefile.html#method-i-find_executable) here.